<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  
    <title>Publications - Styliani-Christina Fragkouli</title>
  

  <meta name="description" content="A Physicist playing with ML & Bioinformatics">
  <meta name="author" content="Styliani-Christina Fragkouli">

  <!-- CSS -->
  <!-- Bootstrap is included in main.css -->
  <link rel="stylesheet" href="/assets/theme/css/main.css">
  <link rel="stylesheet" href="/assets/theme/css/font-awesome.min.css">
  <link rel="stylesheet" href="/assets/theme/css/pygments.css">

  <!-- Fonts -->
  <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700,700i&display=swap" rel="stylesheet">

  <!-- Feeds -->
  <link rel="alternate" type="application/rss+xml" title="Styliani-Christina Fragkouli - RSS feed" href="https://sfragkoul.github.io/feed.xml" >
  <!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-65YGDLCWWV"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-65YGDLCWWV');
</script>

</head>


  <body>
    <nav class="navbar navbar-expand-lg navbar-light bg-light" id="main-nav">
  <div class="container">
    <a class="navbar-brand d-flex align-items-center" href="/">
      
        <img src="/assets/img/groupLogo/telescope.png" class="mr-2">
      
      Styliani-Christina Fragkouli
    </a>
    
      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar-menu">
        Menu <i class="fa fa-bars"></i>
      </button>
      <div class="collapse navbar-collapse" id="navbar-menu">
        <ul class="navbar-nav ml-auto">
        
          
            <li class="navbar-item">
              <a class="nav-link" href="/" >Home</a>
            </li>
          
        
          
            <li class="navbar-item">
              <a class="nav-link" href="/CV/" >CV</a>
            </li>
          
        
          
            <li class="navbar-item">
              <a class="nav-link active" href="/publications/" >Publications</a>
            </li>
          
        
          
            <li class="navbar-item">
              <a class="nav-link" href="/about/" >About</a>
            </li>
          
        
          
            <li class="navbar-item">
              <a class="nav-link" href="/Photography/" >Photography</a>
            </li>
          
        
        </ul>
      </div>
    
  </div>
</nav>


    <main>
      
  <header class="jumbotron bg-image" style="background-image: url('/assets/img/groupLogo/publications.jpg')">

    <div class="container">
      <div class="row">
        <div class="col-md-10 col-lg-8 mx-auto">
          <h1>Publications</h1>
          
          <div class="lead">
</div>
          
          

          <div class="categories">
            
          </div>
        </div>
      </div>
    </div>
  </header>


      <div class="container content">

        <!-- # Bibliography -->

<p><a href="#book-chapters"><strong>Book Chapters</strong></a>   <a href="#journals"><strong>Journals</strong></a>   <a href="#conferences-and-announcements"><strong>Conferences and Announcements</strong></a>   <a href="#other"><strong>Other</strong></a>   <a href="#theses"><strong>Theses</strong></a></p>

<p><br /></p>

<h2 id="book-chapters">Book Chapters</h2>

<p style="margin-bottom:15px"></p>

<ul class="bibliography"><li><div class="text-align: left">
    <span id="Nousi2025">P. Nousi, <b>S.-C. Fragkouli</b>, N. Stergioulas, and A. Tefas, “Deep Learning Methods for Accelerating Gravitational Wave Surrogate Modeling,” in <i>Gravitational Wave Science with Machine Learning</i>, E. Cuoco, Ed. Singapore: Springer Nature Singapore, 2025, pp. 275–288.</span>
</div>

<div id="Nousi2025-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://doi.org/10.1007/978-981-96-1737-1_20" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    
    <li><a data-toggle="collapse" href="#Nousi2025-abstract">Abstract</a></li>
    

    &nbsp; &nbsp;

    
  </ul>

  
  <p id="Nousi2025-abstract" class="collapse">We explore the application of deep learning techniques to accelerate gravitational wave surrogate modeling. We focus on two recent approaches, using artificial neural networks (ANNs) with residual error modeling and autoencoder-driven spiral representation learning. For the ANN method, we demonstrate that adding a second network to learn residual errors significantly improves surrogate model accuracy. The autoencoder approach reveals an inherent spiral structure in the latent space representation of empirical interpolation coefficients. We take advantage of this insight to develop a neural spiral module that can be integrated into network architectures to accelerate training and improve performance. Comprehensive evaluations show that these methods achieve state-of-the-art accuracy while enabling faster waveform generation. The techniques presented have the potential to substantially accelerate gravitational wave data analysis as detector sensitivity improves and event rates increase.</p>
  

</div>
</li></ul>

<p><br /></p>

<h2 id="journals">Journals</h2>

<p style="margin-bottom:15px"></p>

<ul class="bibliography"><li><div class="text-align: left">
    <span id="10.1093/gigascience/giae094">O. A. Attafi, D. Clementel, K. Kyritsis, E. Capriotti, G. Farrell, <b>S.-C. Fragkouli</b>, L. J. Castro, A. Hatos, T. Lenaerts, S. Mazurenko, S. Mozaffari, F. Pradelli, P. Ruch, C. Savojardo, P. Turina, F. Zambelli, D. Piovesan, A. M. Monzon, F. Psomopoulos, and S. C. E. Tosatto, “DOME Registry: implementing community-wide recommendations for reporting supervised machine learning in biology,” <i>GigaScience</i>, vol. 13, p. giae094, Dec. 2024, doi: 10.1093/gigascience/giae094.</span>
</div>

<div id="10.1093/gigascience/giae094-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://doi.org/10.1093/gigascience/giae094" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    
    <li><a data-toggle="collapse" href="#10.1093/gigascience/giae094-abstract">Abstract</a></li>
    

    &nbsp; &nbsp;

    
  </ul>

  
  <p id="10.1093/gigascience/giae094-abstract" class="collapse">Supervised machine learning (ML) is used extensively in biology and deserves closer scrutiny. The Data Optimization Model Evaluation (DOME) recommendations aim to enhance the validation and reproducibility of ML research by establishing standards for key aspects such as data handling and processing, optimization, evaluation, and model interpretability. The recommendations help to ensure that key details are reported transparently by providing a structured set of questions. Here, we introduce the DOME registry (URL: registry.dome-ml.org), a database that allows scientists to manage and access comprehensive DOME-related information on published ML studies. The registry uses external resources like ORCID, APICURON, and the Data Stewardship Wizard to streamline the annotation process and ensure comprehensive documentation. By assigning unique identifiers and DOME scores to publications, the registry fosters a standardized evaluation of ML methods. Future plans include continuing to grow the registry through community curation, improving the DOME score definition and encouraging publishers to adopt DOME standards, and promoting transparency and reproducibility of ML in the life sciences.</p>
  

</div>
</li>
<li><div class="text-align: left">
    <span id="doi:10.1080/17460913.2024.2400853"><b>S.-C. Fragkouli</b>, D. Solanki, L. Castro, F. Psomopoulos, N. Queralt-Rosinach, D. Cirillo, and L. Crossman, “Synthetic data: how could it be used in infectious disease research?,” <i>Future Microbiology</i>, vol. 0, no. 0, pp. 1–6, 2024, doi: 10.1080/17460913.2024.2400853.</span>
</div>

<div id="doi:10.1080/17460913.2024.2400853-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://doi.org/10.1080/17460913.2024.2400853" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="FRAGKOULI2023110746"><b>S.-C. Fragkouli</b>, P. Nousi, N. Passalis, P. Iosif, N. Stergioulas, and A. Tefas, “Deep residual error and bag-of-tricks learning for gravitational wave surrogate modeling,” <i>Applied Soft Computing</i>, p. 110746, 2023, doi: https://doi.org/10.1016/j.asoc.2023.110746.</span>
</div>

<div id="FRAGKOULI2023110746-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://www.sciencedirect.com/science/article/pii/S1568494623007640" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    
    <li><a data-toggle="collapse" href="#FRAGKOULI2023110746-abstract">Abstract</a></li>
    

    &nbsp; &nbsp;

    
  </ul>

  
  <p id="FRAGKOULI2023110746-abstract" class="collapse">Deep learning methods have been employed in gravitational-wave astronomy to accelerate the construction of surrogate waveforms for the inspiral of spin-aligned black hole binaries, among other applications. We face the challenge of modeling the residual error of an artificial neural network that models the coefficients of the surrogate waveform expansion (especially those of the phase of the waveform) which we demonstrate has sufficient structure to be learnable by a second network. Adding this second network, we were able to reduce the maximum mismatch for waveforms in a validation set by 13.4 times. We also explored several other ideas for improving the accuracy of the surrogate model, such as the exploitation of similarities between waveforms, the augmentation of the training set, the dissection of the input space, using dedicated networks per output coefficient and output augmentation. In several cases, small improvements can be observed, but the most significant improvement still comes from the addition of a second network that models the residual error. Since the residual error for more general surrogate waveform models (when e.g., eccentricity is included) may also have a specific structure, one can expect our method to be applicable to cases where the gain in accuracy could lead to significant gains in computational time.</p>
  

</div>
</li>
<li><div class="text-align: left">
    <span id="Zaragoza_Infante_2022">L. Zaragoza-Infante, V. Junet, N. Pechlivanis, <b>S.-C. Fragkouli</b>, S. Amprachamian, T. Koletsa, A. Chatzidimitriou, M. Papaioannou, K. Stamatopoulos, A. Agathangelidis, and F. Psomopoulos, “IgIDivA: immunoglobulin intraclonal diversification analysis,” <i>Briefings in Bioinformatics</i>, Aug. 2022, doi: 10.1093/bib/bbac349.</span>
</div>

<div id="Zaragoza_Infante_2022-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://doi.org/10.1093%2Fbib%2Fbbac349" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    
    <li><a data-toggle="collapse" href="#Zaragoza_Infante_2022-abstract">Abstract</a></li>
    

    &nbsp; &nbsp;

    
  </ul>

  
  <p id="Zaragoza_Infante_2022-abstract" class="collapse">Intraclonal diversification (ID) within the immunoglobulin (IG) genes expressed by B cell clones arises due to ongoing somatic hypermutation (SHM) in a context of continuous interactions with antigen(s). Defining the nature and order of appearance of SHMs in the IG genes can assist in improved understanding of the ID process, shedding light into the ontogeny and evolution of B cell clones in health and disease. Such endeavor is empowered thanks to the introduction of high-throughput sequencing in the study of IG gene repertoires. However, few existing tools allow the identification, quantification and characterization of SHMs related to ID, all of which have limitations in their analysis, highlighting the need for developing a purpose-built tool for the comprehensive analysis of the ID process. In this work, we present the immunoglobulin intraclonal diversification analysis (IgIDivA) tool, a novel methodology for the in-depth qualitative and quantitative analysis of the ID process from high-throughput sequencing data. IgIDivA identifies and characterizes SHMs that occur within the variable domain of the rearranged IG genes and studies in detail the connections between identified SHMs, establishing mutational pathways. Moreover, it combines established and new graph-based metrics for the objective determination of ID level, combined with statistical analysis for the comparison of ID level features for different groups of samples. Of importance, IgIDivA also provides detailed visualizations of ID through the generation of purpose-built graph networks. Beyond the method design, IgIDivA has been also implemented as an R Shiny web application. IgIDivA is freely available at https://bio.tools/igidiva</p>
  

</div>
</li>
<li><div class="text-align: left">
    <span id="NOUSI202267">P. Nousi, <b>S.-C. Fragkouli</b>, N. Passalis, P. Iosif, T. Apostolatos, G. Pappas, N. Stergioulas, and A. Tefas, “Autoencoder-driven spiral representation learning for gravitational wave surrogate modelling,” <i>Neurocomputing</i>, vol. 491, pp. 67–77, 2022, doi: https://doi.org/10.1016/j.neucom.2022.03.052.</span>
</div>

<div id="NOUSI202267-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://www.sciencedirect.com/science/article/pii/S0925231222003472" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    
    <li><a data-toggle="collapse" href="#NOUSI202267-abstract">Abstract</a></li>
    

    &nbsp; &nbsp;

    
  </ul>

  
  <p id="NOUSI202267-abstract" class="collapse">Recently, artificial neural networks have been gaining momentum in the field of gravitational wave astronomy, for example in surrogate modelling of computationally expensive waveform models for binary black hole inspiral and merger. Surrogate modelling yields fast and accurate approximations of gravitational waves and neural networks have been used in the final step of interpolating the coefficients of the surrogate model for arbitrary waveforms outside the training sample. We investigate the existence of underlying structures in the empirical interpolation coefficients using autoencoders. We demonstrate that when the coefficient space is compressed to only two dimensions, a spiral structure appears, wherein the spiral angle is linearly related to the mass ratio. Based on this finding, we design a spiral module with learnable parameters, that is used as the first layer in a neural network, which learns to map the input space to the coefficients. The spiral module is evaluated on multiple neural network architectures and consistently achieves better speed-accuracy trade-off than baseline models. A thorough experimental study is conducted and the final result is a surrogate model which can evaluate millions of input parameters in a single forward pass in under 1 ms on a desktop GPU, while the mismatch between the corresponding generated waveforms and the ground-truth waveforms is better than the compared baseline methods. We anticipate the existence of analogous underlying structures and corresponding computational gains also in the case of spinning black hole binaries.</p>
  

</div>
</li></ul>

<p><br /></p>

<h2 id="conferences-and-announcements">Conferences and Announcements</h2>
<p style="margin-bottom:15px"></p>

<ul class="bibliography"></ul>
<ul class="bibliography"><li><div class="text-align: left">
    <span id="fragkouli_2024_14186510"><b>S.-C. Fragkouli</b>, N. Pechlivanis, A. Anastasiadou, G. Karakatsoulis, A. Orfanou, P. Kollia, A. Agathangelidis, and F. Psomopoulos, “Synth4bench: generating synthetic genomics data for the evaluation of somatic variant callers, <em>23rd European Conference on Computational Biology (ECCB2024)</em>,” Nov. 2024, doi: 10.5281/zenodo.14186510.</span>
</div>

<div id="fragkouli_2024_14186510-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://doi.org/10.5281/zenodo.14186510" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="synth4bench_HSCBB23"><b>S.-C. Fragkouli</b>, N. Pechlivanis, A. Orfanou, A. Anastasiadou, A. Agathangelidis, and F. Psomopoulos, “Synth4bench: a framework for generating synthetic genomics data for the evaluation of somatic variant calling algorithms, <em>17th Conference of Hellenic Society for Computational Biology and Bioinformatics (HSCBB)</em>,” Oct. 2023, doi: 10.5281/zenodo.8432060.</span>
</div>

<div id="synth4bench_HSCBB23-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://doi.org/10.5281/zenodo.8432060" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="synthECCB23"><b>S.-C. Fragkouli</b>, N. Pechlivanis, A. Agathangelidis, and F. Psomopoulos, “Synthetic Genomics Data Generation and Evaluation for the Use Case of Benchmarking Somatic Variant Calling Algorithms, 31st Conference in Intelligent Systems For Molecular Biology and the 22nd European Conference On Computational Biology (ISΜB-ECCB23)  ,” Jul. 2023, doi: 10.7490/f1000research.1119575.1.</span>
</div>

<div id="synthECCB23-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://doi.org/10.7490/f1000research.1119575.1" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="scfeaECCB23">G. Gavriilidis, <b>S.-C. Fragkouli</b>, E. Theodosiou, V. Vasileiou, S. Keisaris, and F. Psomopoulos, “SCell-wise fluxomics of Chronic Lymphocytic Leukemia single-cell data reveal novel metabolic adaptations to Ibrutinib therapy, 31st Conference in Intelligent Systems For Molecular Biology and the 22nd European Conference On Computational Biology (ISΜB-ECCB23)  ,” Jul. 2023, doi: 10.13140/RG.2.2.14185.26720.</span>
</div>

<div id="scfeaECCB23-materials">
  <ul class="nav nav-pills">
    
    <li><a href="http://dx.doi.org/10.13140/RG.2.2.14185.26720" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="HSCBB22"><b>S.-C. Fragkouli</b>, A. Agathangelidis, and F. Psomopoulos, “Shedding Light on Somatic Variant Calling, 16th Conference of the Hellenic Society for Computational Biology and Bioinformatics HSCBB22,” Oct. 2022, doi: 10.13140/RG.2.2.12701.18402.</span>
</div>

<div id="HSCBB22-materials">
  <ul class="nav nav-pills">
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li></ul>

<p><br /></p>

<h2 id="other">Other</h2>
<p style="margin-bottom:15px"></p>

<ul class="bibliography"><li><div class="text-align: left">
    <span id="Fragkouli2024.03.07.582313__paper_preprint"><b>S.-C. Fragkouli</b>, N. Pechlivanis, A. Anastasiadou, G. Karakatsoulis, A. Orfanou, P. Kollia, A. Agathangelidis, and F. E. Psomopoulos, “Synth4bench: Synthetic Data Generation for Benchmarking Tumor-Only Somatic Variant Calling Algorithms.” 2025, doi: 10.1101/2024.03.07.582313.</span>
</div>

<div id="Fragkouli2024.03.07.582313__paper_preprint-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://www.biorxiv.org/content/10.1101/2024.03.07.582313v2" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="25SynthDatasets"><b>S.-C. Fragkouli</b>, A. Agathangelidis, and F. E. Psomopoulos, “25 Synthetic TP53 Genomic Datasets for Benchmarking and Method Development.” Oct. 2025, doi: 10.5281/zenodo.16524193.</span>
</div>

<div id="25SynthDatasets-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://zenodo.org/records/16524193" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="fragkouli2025elixirscopingreviewdomainspecific"><b>S.-C. Fragkouli</b>, S. Iqbal, L. Crossman, B. Gravel, N. Masued, M. Onders, D. Haseja, A. Stikkelman, A. Valencia, T. Lenaerts, F. Psomopoulos, P. Ó. Broin, N. Queralt-Rosinach, and D. Cirillo, “An ELIXIR scoping review on domain-specific evaluation metrics for synthetic data in life sciences.” 2025, [Online]. Available at: https://arxiv.org/abs/2506.14508.</span>
</div>

<div id="fragkouli2025elixirscopingreviewdomainspecific-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://arxiv.org/abs/2506.14508" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="farrell2025opensustainableaichallenges">G. Farrell <i>et al.</i>, “Open and Sustainable AI: challenges, opportunities and the road ahead in the life sciences.” 2025, [Online]. Available at: https://arxiv.org/abs/2505.16619.</span>
</div>

<div id="farrell2025opensustainableaichallenges-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://arxiv.org/abs/2505.16619" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="CAN.Heal_Brussels"><b>S.-C. Fragkouli</b> and F. E. Psomopoulos, “WP11 –NGS INCLUDING LIQUID BIOPSY,” <i>CAN.Heal consortium meeting</i>. Oct. 2024, doi: 10.5281/zenodo.13959394.</span>
</div>

<div id="CAN.Heal_Brussels-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://zenodo.org/records/13959394" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="attafi2024domeregistryimplementingcommunitywide__paper_preprint">O. A. Attafi, D. Clementel, K. Kyritsis, E. Capriotti, G. Farrell, <b>S.-C. Fragkouli</b>, L. J. Castro, A. Hatos, T. Lenaerts, S. Mazurenko, S. Mozaffari, F. Pradelli, P. Ruch, C. Savojardo, P. Turina, F. Zambelli, D. Piovesan, A. M. Monzon, F. Psomopoulos, and S. C. E. Tosatto, “DOME Registry: Implementing community-wide recommendations for reporting supervised machine learning in biology.” 2024, [Online]. Available at: https://arxiv.org/abs/2408.07721.</span>
</div>

<div id="attafi2024domeregistryimplementingcommunitywide__paper_preprint-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://arxiv.org/abs/2408.07721" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="MLFGAHM24">F. Psomopoulos, E. Capriotti, N. Rosinach, D. Cirillo, L. Castro, S. Tosatto, and <b>the ELIXIR ML Focus Group members</b>, “The impact of the ELIXIR community in Machine Learning.” ELIXIR All Hands Meeting, Jun. 2024, doi: 10.7490/f1000research.1119794.1.</span>
</div>

<div id="MLFGAHM24-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://f1000research.com/posters/13-797" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="FragkouliAHM24"><b>S.-C. Fragkouli</b>, N. Pechlivanis, A. Anastasiadou, G. Karakatsoulis, A. Orfanou, P. Kollia, A. Agathangelidis, and F. E. Psomopoulos, “Exploring Somatic Variant Callers’ Behavior:  A Synthetic Genomics Feature Space Approach.” ELIXIR All Hands Meeting, Jun. 2024, doi: 10.7490/f1000research.1119793.1.</span>
</div>

<div id="FragkouliAHM24-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://f1000research.com/posters/13-795" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="fragkouli2024syntheticdatausedinfectious__paper_preprint"><b>S.-C. Fragkouli</b>, D. Solanki, L. J. Castro, F. E. Psomopoulos, N. Queralt-Rosinach, D. Cirillo, and L. C. Crossman, “Synthetic data: How could it be used for infectious disease research?” 2024, [Online]. Available at: https://arxiv.org/abs/2407.06211.</span>
</div>

<div id="fragkouli2024syntheticdatausedinfectious__paper_preprint-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://arxiv.org/abs/2407.06211" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="CAN.Heal_Paris"><b>S.-C. Fragkouli</b> and F. E. Psomopoulos, “WP11 –NGS INCLUDING LIQUID BIOPSY,” <i>CAN.Heal consortium meeting</i>. Apr. 2024, doi: 10.5281/zenodo.13843594.</span>
</div>

<div id="CAN.Heal_Paris-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://zenodo.org/records/13843595" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="10SynthDatasets"><b>S.-C. Fragkouli</b>, A. Agathangelidis, and F. E. Psomopoulos, “10 Synthetic Genomics Datasets.” Feb. 2024, doi: 10.5281/zenodo.10683211.</span>
</div>

<div id="10SynthDatasets-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://zenodo.org/records/10683211" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="elixireuropeRDMkit">F. Adriano, E. Parkinson, D. Bianchini, F. Psomopoulos, M. Varadi, M. Andrabi, <b>S.-C. Fragkouli</b>, and U. Vadadokhau, “RDMkit, Your Domain, Machine Learning.” 2024, [Online]. Available at: https://rdmkit.elixir-europe.org/machine_learning.</span>
</div>

<div id="elixireuropeRDMkit-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://rdmkit.elixir-europe.org/machine_learning" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="TP53synthdata"><b>S.-C. Fragkouli</b>, A. Agathangelidis, and F. E. Psomopoulos, “TP53 synthetic genomics data for benchmarking variant callers.” Jun. 2023, doi: 10.5281/zenodo.8095898.</span>
</div>

<div id="TP53synthdata-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://zenodo.org/records/8095898" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="DOME"><b>S.-C. Fragkouli</b>, “DOME ANNOTATION: THE CROWDSOURCING EFFORT,” <i>ELIXIR DOME Strategic Implementation Study Kickoff meeting</i>. Apr. 2023, doi: 10.5281/zenodo.10688491.</span>
</div>

<div id="DOME-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://zenodo.org/records/10688491" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="lineagespot">N. Pechlivanis, M. Tsagiopoulou, M. C. Maniou, A. Togkousidis, E. Mouchtaropoulou, T. Chassalevris, S. Chaintoutis, C. Dovas, M. Petala, M. Kostoglou, T. Karapantsios, S. Laidou, E. Vlachonikola, A. Orfanou, <b>S.-C. Fragkouli</b>, S. Keisaris, A. Chatzidimitriou, A. Papadopoulos, N. Papaioannou, A. Argiriou, and F. E. Psomopoulos, “Detection of SARS-CoV-2 lineages in wastewater samples using next-generation sequencing.” 2022, doi: https://doi.org/10.18129/B9.bioc.lineagespot.</span>
</div>

<div id="lineagespot-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://bioconductor.org/packages/release/bioc/html/lineagespot.html" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="https://doi.org/10.48550/arxiv.2203.08434"><b>S.-C. Fragkouli</b>, P. Nousi, N. Passalis, P. Iosif, N. Stergioulas, and A. Tefas, “Deep Residual Error and Bag-of-Tricks Learning for Gravitational Wave Surrogate Modeling.” arXiv, 2022, doi: 10.48550/ARXIV.2203.08434.</span>
</div>

<div id="https://doi.org/10.48550/arxiv.2203.08434-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://arxiv.org/abs/2203.08434" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="VirgoPresentation"><b>S.-C. Fragkouli</b>, “Deep Residual Error and Bag-of-Tricks Learning for Gravitational Wave Surrogate Μodeling,” <i>LIGO-Virgo-Kagra Collaboration Waveform WG meeting</i>. Feb. 2022, doi: 10.5281/zenodo.10688323.</span>
</div>

<div id="VirgoPresentation-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://zenodo.org/records/10688323" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="G2net"><b>S.-C. Fragkouli</b>, P. Nousi, N. Passalis, P. Iosif, N. Stergioulas, and A. Tefas, “Deep Residual Error and Bag-of-Tricks Learning for Gravitational Wave Surrogate Modeling,” <i>G2net WG1(Machine Learning for Gravitational Wave Astronomy) meeting</i>. Apr. 2022.</span>
</div>

<div id="G2net-materials">
  <ul class="nav nav-pills">
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li></ul>

<p><br /></p>

<h2 id="theses">Theses</h2>

<ul class="bibliography"></ul>
<ul class="bibliography"><li><div class="text-align: left">
    <span id="thesis2021"><b>S.-C. Fragkouli</b>, “Deep Learning Applications on Gravitational Waves.” Informatics Department, Aristotle University of Thessaloniki, 2021, [Online]. Available at: https://ikee.lib.auth.gr/record/335161.</span>
</div>

<div id="thesis2021-materials">
  <ul class="nav nav-pills">
    
    <li><a href="https://ikee.lib.auth.gr/record/335161" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li>
<li><div class="text-align: left">
    <span id="thesis2017"><b>S.-C. Fragkouli</b>, “Study of the merging of a neutron star with a black hole and the role of the equation of state.” Physics Department, Aristotle University of Thessaloniki, 2017, [Online]. Available at: http://ikee.lib.auth.gr/record/294627/files/thesis.pdf.</span>
</div>

<div id="thesis2017-materials">
  <ul class="nav nav-pills">
    
    <li><a href="http://ikee.lib.auth.gr/record/294627/files/thesis.pdf" target="_blank">Available online</a></li>
    

    &nbsp; &nbsp;

    

    &nbsp; &nbsp;

    
  </ul>

  

</div>
</li></ul>



      </div>
    </main>

    <footer>
  <div class="container">
    <div class="row">
      <div class="col-lg">
        <ul class="list-inline">
          
            <li class="list-inline-item">
              <a href="mailto:scfragkouli@gmail.com">
                <span class="fa-stack fa-lg">
                  <i class="fa fa-circle fa-stack-2x"></i>
                  <i class="fa fa-envelope-o fa-stack-1x fa-inverse"></i>
                </span>
              </a>
            </li>
          

          

          
            <li class="list-inline-item">
              <a href="https://github.com/sfragkoul">
                <span class="fa-stack fa-lg">
                  <i class="fa fa-circle fa-stack-2x"></i>
                  <i class="fa fa-github fa-stack-1x fa-inverse"></i>
                </span>
              </a>
            </li>
          
        </ul>
      </div>

      
      <div class="col-lg">
        <p>This is my personal website. <br /> <a href="https://github.com/sfragkoul">My GitHub</a></p>


        

      </div>
      
    </div>

    <div class="small mt-2">
      
        <p>© Styliani-Christina Fragkouli</p>
      
      
        <p>Content on this site is licensed under a <a href="http://creativecommons.org/licenses/by/4.0/">CC BY 4.0 License</a>. | Theme: <a href="https://github.com/peterdesmet/petridish">Petridish</a></p>

      
    </div>
  </div>
</footer>

<script src="/assets/theme/js/jquery.min.js"></script>
<script src="/assets/theme/js/popper.min.js"></script>
<script src="/assets/theme/js/bootstrap.min.js"></script>
<script src="/assets/theme/js/anchor.min.js"></script>

<script>
  // Show anchors when hovering over headings, using anchor.js
  (function () {
    const headings = ".content h2[id], .content h3[id], .content h4[id], .content h5[id], .content h6[id]";
    anchors.options = { placement: "right" };
    anchors.add(headings);
  })();
</script>



  </body>
</html>
